app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,8 @@
+class CustomerItemsController < ApplicationController
+    def create 
+        @customer = Customer.find(params[:id])
+        item_of_customer = Item.find_by(id: params[:item_id])
+        CustomerItem.create(customer_id: @customer.id, item_id: item_of_customer.id)
```

Comment:
You don't necessarily need to find the customer and the item objects before line 5, as you can just use the params to create the CustomerItem

Example: `CustomerItem.create(customer_id: params[:id], item_id: params[:item_id])`